tests/testthat/test-bundle.R

```diff
@@ -644,6 +644,38 @@ test_that("tarImplementation: checks environment variable and option before usin
 })
 
 
+# tweakRProfile -----------------------------------------------------------
+
+test_that(".Rprofile tweaked automatically", {
```

Comment:
Could you confirm that `.Rprofile` files without the patterns are left untouched?